
From: David Harder <david@davidjharder.ca>
Date: Mon, 16 May 2022 19:08:32 -0500
Subject: [PATCH] Change prefix

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 4336bb8..ba2a22f 100644
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,6 @@
 CC ?= clang
 CFLAGS ?= -Og -Wall -Wpedantic -Wextra
-PREFIX ?= /usr/local/
+PREFIX ?= /usr/
 BINDIR ?= bin
 DESTDIR ?=
 

